lib/src/configuration.dart

```diff
@@ -39,14 +39,19 @@ class Configuration {
   /// [readOnly] controls whether a [Realm] is opened as readonly.
   /// This allows opening it from locked locations such as resources,
   /// bundled with an application.  The realm file must already exists.
-  Configuration(List<SchemaObject> schemaObjects, {bool readOnly = false})
+  ///
+  /// If [inMemoryOnly] is set, then the [Realm] instance will not be persisted, and hence be _deleted_ after it is closed.
```

Comment:
I think other SDKs have different doc here.

Reply:
Could you please check this comment: https://github.com/realm/realm-dart/pull/280#r817680594

Reply:
add a review with a suggestion

Reply:
Done.